.github/workflows/ci.yml

```diff
@@ -56,8 +56,6 @@ jobs:
           - classic-test-app
           - test-app
         test-suite:
-          - test:one ember-3.0
-          - test:one ember-lts-3.4
```

Comment:
Why does this change? Can we not support 3.0 and 3.4 when using QUnit?

Reply:
Sorry, I wanted to write ember-qunit 5. ember-qunit 5 supports ember 3.8+. 
https://github.com/emberjs/ember-qunit

Reply:
Can we use a version of ember-qunit that supports Ember 3.0 as well? I'd rather upgrade to ember-qunit 5 (and drop support for Ember < 3.8) in a separate PR then.

Reply:
Yes, I can use ember-qunit 4 with some change in the source code. I will do it.